doc/user_guide/eels.rst

```diff
@@ -159,6 +159,19 @@ Furthermore, the components are available in the user namespace
     >>> background
     <background (PowerLaw component)>
 
+.. warning::
+
+   This feature is deprecated and will be removed in HyperSpy 0.9. To access
+   the automatically created component use the following equivalent syntax:
+
+    .. code-block:: python
+
+        >>> m.components.N_K
+        <N_K (EELSCLEdge component)>
+        >>> m.components.B_K
+        <B_K (EELSCLEdge component)>
+        >>> m.background
+        <background (PowerLaw component)>
```

Comment:
`m.components.background` ? 

I think it would be strange to single out this one particular component..